README.md

```diff
@@ -1,11 +1,37 @@
 # SecretHub Kubernetes Mutating Webhook
 
-This mutating webhook allows you to use secret references (`secrethub://path/to/secret`) in any containers spec, without including SecretHub in the image itself.
+This mutating webhook allows you to use secret references (`secrethub://path/to/secret`) in any containers spec, without including SecretHub in the image itself:
 
-It detects whether the container environment contains secret references (`secrethub://path/to/secret`) and when it does:
-1. It creates a volume which will hold the SecretHub CLI.
-1. It creates an init container which copies the SecretHub CLI into the volume.
-1. It mounts the volume to the target container.
-1. It prefixes the target containers command with `<path/to/volume>/secrethub run --`.
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: my-app
+  annotations:
+    secrethub.io/mutate: my-app
+spec:
+  containers:
+    - name: my-app
+      image: my-image
+      env:
+        - name: STRIPE_SECRET_KEY
+          value: secrethub://acme/app/prod/stripe/secret_key
+        - name: PGPASSWORD
+          value: secrethub://acme/app/prod/pg/password
+```
+
+You can annotate your pod spec with `secrethub.io/mutate` which expects a comma separated list of the names of the containers to mutate.
+
+When the annotation is found:
+- A volume which will hold the SecretHub CLI is created.
+- An init container which copies the SecretHub CLI into the volume is created.
+
+And for every container that is listed in the `secrethub.io/mutate` annotation:
+- The volume is mounted to the container.
+- The command is prefixed with `<path/to/volume>/secrethub run --`.
+
+The version of the SecretHub CLI to use can optionally be configured with `secrethub.io/version`. If it is not set, the `latest` version is used.
```

Comment:
This may be very logical for us, but I would explicitly state that the version should be in the form `v1.2.3`. I think this can most easily be tackled by giving an example of the annotation.

Reply:
Actually, it should be without the `v`, all the more reason to document it :+1: